docs/reference/indices/aliases.asciidoc

```diff
@@ -30,138 +35,124 @@ POST /_aliases
+The object body contains options for the alias. Supports an empty object.
 +
-NOTE: You cannot add <<data-streams,data streams>> to an index alias.
-
+.Properties of `<action>`
+[%collapsible%open]
+=====
 `alias`::
-(String)
-Comma-separated list or wildcard expression of index alias names to add, remove,
-or delete. Supports <<date-math-index-names,date math>>.
-+
-If the `aliases` parameter is not specified,
-this parameter is required for the `add` or `remove` action.
+(Required*, string) Alias for the action. Index alias names support
+<<date-math-index-names,date math>>. If `aliases` is not specified, the `add`
+and `remove` actions require this parameter. The `remove_index` action doesn't
+support this parameter.
 
 `aliases`::
-(Array of strings)
-Array of index alias names to add, remove, or delete. Supports
-<<date-math-index-names,date math>>.
-+
-If the `alias` parameter is not specified, this parameter is required for the
-`add` or `remove` action.
+(Required*, array of strings) Aliases for the action. Index alias names support
+<<date-math-index-names,date math>>. If `alias` is not specified, the `add` and
+`remove` actions require this parameter. The `remove_index` action doesn't
+support this parameter.
 
+// tag::alias-options[]
 `filter`::
-(Optional, query object)
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index-alias-filter]
+(Optional, <<query-dsl,Query DSL object>> Query used to limit documents the
+alias can access. Data stream aliases don't support this parameter.
+// end::alias-options[]
 +
-See <<filter-alias>> for an example.
+Only the `add` action supports this parameter.
 
-`is_hidden`::
-(Optional, Boolean)
-If `true`, the alias will be excluded from wildcard expressions by default,
-unless overridden in the request using the `expand_wildcards` parameter,
-similar to <<index-hidden,hidden indices>>. This property must be set to the
-same value on all indices that share an alias. Defaults to `false`.
+`index`::
+(Required*, string) Data stream or index for the action. Supports wildcards
+(`*`). Wildcard patterns that match both data streams and indices return an
+error. If `indices` is not specified, this parameter is required.
 
-`must_exist`::
-(Optional, Boolean)
-If `true`, the alias to remove must exist. Defaults to `false`.
+`indices`::
+(Required*, array of strings) Data streams or indices for the action. Supports
+wildcards (`*`). Wildcard patterns that match both data streams and indices
+return an error. If `index` is not specified, this parameter is required.
 
-`is_write_index`::
-(Optional, Boolean)
-If `true`, assigns the index as an alias's write index.
-Defaults to `false`.
-+
-An alias can have one write index at a time.
+// tag::alias-options[]
+`index_routing`::
+(Optional, string) Value used to route indexing operations to a specific shard.
+If specified, this overwrites the `routing` value for indexing operations. Data
+stream aliases don't support this parameter.
```

Comment:
This line can now be removed. I just merged #73462.

Reply:
Thanks. I've removed the line for the `is_write_index` parameter.